### DISCUSSION TITLE
Stay alternative: set an application to a display

### DISCUSSION THREAD

**Discussion by mcnahum** [rxhanson/RectanglePro-Community#620] [Q&A]
Hello,
I'm using 3 displays and I like to have an application to always open to the same display:
Outlook on the right one, Terminal on the bottom one, Virtualbuddy on left one , ....

For a long time I was using the "stay" application but it use the "number" of the display who is not something who is fixed ( 2 and 3 always changing at every wake up or reboot and I didn't found a way to fix it)

The application "stay" is not supported and you can find in reddit a lot of people looking for an alternative.
one Recommended Rectangle Pro ... but after few testing I don't find that option. ...

1°/ I missied it, please let me know where it is and take my money for licenses...
2°/ it is not there,  please take it as a feature request where a lot of customers will love ( look at reddit :-> )

**Comment by rxhanson** [comment]
Rectangle Pro isn't a drop-in replacement for Stay at this time, but you can save a layout of windows and recall that layout via keyboard shortcut, wake, or display connection/disconnection. 
* Here's a quick guide for it: https://github.com/rxhanson/RectanglePro-Community/discussions/111
* Saving a layout will store window sizes and positions in absolute values, as in specific values in all your displays set in one big 2D coordinate space. This means that if a display is not available, then a window will not be sent to that position. 
* You can change to relative values for sizes and positions by setting values less than 1 (1/2, 0.5, etc). When doing that you can specify which display you want them sent to, and Rectangle Pro will number displays based on the display's current location in 2D space. The display in the top left most position will be 1 and bottom right most position will be the largest number.
* There's a checkbox for "Launch closed/minimized apps" which will launch an app and apply the layout, but there's not currently functionality that will automatically apply a size/position to a newly opened window from outside of the app. I may add this in at some point, but the reason I have held off is because it requires constantly polling active windows.
* There are some displays that throw off timing of applying layouts automatically upon display connection/disconnection, preventing it from happening automatically. Unfortunately, I don't have a good solution for that at this time.
